pkg/api/resolver_mutation_performer.go

```diff
@@ -83,6 +84,24 @@ func (r *mutationResolver) PerformerCreate(ctx context.Context, input models.Per
 	} else {
 		newPerformer.Favorite = sql.NullBool{Bool: false, Valid: true}
 	}
+	if input.Details != nil {
+		newPerformer.Details = sql.NullString{String: *input.Details, Valid: true}
+	}
+	if input.Deathdate != nil {
+		newPerformer.Deathdate = models.SQLiteDate{String: *input.Deathdate, Valid: true}
+
+		if *input.Deathdate != "" {
+			if utils.IfDateHigherThanAnotherDate(*input.Birthdate, *input.Deathdate) {
+				return nil, errors.New("The date of Death should be higher than the date of Birth")
```

Comment:
Declare an error at the top of the file and use that instead. 

Error message should be all lowercase.